app/styles/_core.less

```diff
@@ -98,8 +98,15 @@
     font-style: normal;
     margin-top: -5px;
   }
+  .ui-select-match-text > span {
+      width: 85%;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      position: absolute;
+  }
+
   .btn {
-    padding-right: 25px; // prevent text from showing behind select arrow
+    padding-right: 10px; // prevent text from showing behind select arrow
```

Comment:
yep, I've take care about that with adding 
```
  .ui-select-match-text > span {
      width: 85%;
      overflow: hidden;
      text-overflow: ellipsis;
      position: absolute;
  }
```
so the text will be cropped and wont overflow the selectbox.